spec/spec_helper.rb

```diff
@@ -28,6 +28,8 @@
     c.syntax = :expect
   end
 
+  config.alias_it_should_behave_like_to :includes_mixin, 'includes mixin:'
```

Comment:
I am honestly not in the mood for fighting this, so whatever, I will change it.

But just for the record, here is why I didn't use `include_examples` from the get go.  From [the rspec docs](http://www.relishapp.com/rspec/rspec-core/v/3-5/docs/example-groups/shared-examples):

```
include_examples "name"      # include the examples in the current context
it_behaves_like "name"       # include the examples in a nested context
it_should_behave_like "name" # include the examples in a nested context
matching metadata            # include the examples in the current context
```

Since I used `before(:context)`/`after(:context)` in the top level here, using `include_examples` will break other tests this is include with if they rely on a clean DB state to do there tests (which at least one of them does).

`it_behaves_like "miq ownership"` sounds like butt...